### ISSUE TITLE
RKE2 Cluster system agent upgrader failed mount

### ISSUE THREAD

**Issue by rmweir** [rancher/rancher#34494]
**Rancher Server Setup**
- Rancher version: 6af91dae03f434e171caf450e3769528e2788e0a
- Installation option (Docker install/Helm Chart):
docker install

**Information about the Cluster**
- Kubernetes version: v1.21.4+rke2r1
- Cluster Type (Local/Downstream):
  Downstream
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider):
    RKE2

**Describe the bug**

Last event after provision RKE2 cluster failed.

**To Reproduce**
Provision RKE2 cluster on AWS

**Result**
Last even shows "FailedMount" in "Reason" column.

**Expected Result**
Events should succeed.

**Screenshots**

![image](https://user-images.githubusercontent.com/19376037/131013653-83fc2acf-f513-488e-8596-2963de08e589.png)

**Comment by snasovich** [comment]
Leaving it open due to issue linked above.

**Comment by snasovich** [comment]
Moving to test as this shouldn't be an issue any longer. QA to watch for it to reproduce while testing 2.6.5.

**Comment by Josh-Diamond** [comment]
Ticket #34494 - Test Results

With Docker on a single-node instance:

Verified on rancher `v2.6-457ba7ce4c4d5ddbe1abe82533e88c7d849d139c-head`:
1. Fresh install of rancher `v2.6-head`
2. As admin, provision a downstream RKE2 Node driver cluster [ I tested this with AWS ]
3. Verified - RKE2 Node driver cluster successfully provisions [provisioning took 8 minutes]

Screenshot:

`AWS Node driver RKE2`
<img width="1549" alt="Screen Shot 2022-04-01 at 3 40 27 PM" src="https://user-images.githubusercontent.com/46494969/161351313-a09c3391-a2c6-4814-8479-f820c2cb26ac.png">
